src/components/molecules/AnywayHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent, useMemo, CSSProperties } from "react";
+import { AnywayAppBar } from "../atoms/AnywayAppBar";
+import { AnywayLogo } from "../atoms/AnywayLogo";
+import { SignInIcon } from "../atoms/SignInIcon";
+
+export const AnywayHeader: FunctionComponent = () => {
+  const appBarStyle = useMemo<CSSProperties>(
```

Comment:
No reason to memoise this, it's a constant.

Also, because it's a constant, I'd move it outside to a constant at the top of the file:
```
// imports here

 const STYLES = {
  backgroundColor: "#fafafa",
  height: "6vh",
  display: "flex",
  flexDirection: "row",
  alignItems: "center",
  justifyContent: "space-between",
  paddingLeft: "2vw",
  paddingRight: "2vw",
}

export const AnywayHeader: FunctionComponent = () = {
...

Reply:
Also, please read about MaterialUI's `useStyles` hook.